website/redirects.next.js

```diff
@@ -14,6 +14,11 @@ module.exports = [
     destination: '/intro/getting-started',
     permanent: false,
   },
+  {
+    source: '/api',
+    destination: '/api-docs',
+    permanent: true,
```

Comment:
I'll leave it up to the reviewers to determine if this should be a permanent redirect or not.